stdlib/public/SDK/XCTest/XCTest.swift

```diff
@@ -1030,6 +1030,28 @@ public func XCTAssertThrowsError<T>(_ expression: @autoclosure () throws -> T, _
   }
 }
 
+public func XCTAssertNoThrow<T>(_ expression: @autoclosure () throws -> T, _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line) {
+  let assertionType = _XCTAssertionType.assertion_NoThrow
+
+  let result = _XCTRunThrowableBlock { _ = try expression() }
+
+  switch result {
+  case .success:
+    guard let error = error else {
```

Comment:
Re-iterating because indent changes collapsed my comment:

> This code is dead (and won't compile). Can you remove it and make this case simply return?